lib/npmdc/errors.rb

```diff
@@ -69,18 +80,33 @@ def banner
       end
     end
 
+    class NoYarnLockFileError < CheckerError
```

Comment:
And it also would be great to extract checker-specific errors to separate modules.